src/test/java/com/bobocode/svydovets/context/ApplicationContext.java

```diff
@@ -0,0 +1,17 @@
+package com.bobocode.svydovets.context;
+
+import com.bobocode.svydovets.exception.NoSuchBeanDefinitionException;
+import com.bobocode.svydovets.exception.NoUniqueBeanDefinitionException;
+
+import java.util.Map;
+
+public interface ApplicationContext {
+
+    <T> T getBean(Class<T> beanType) throws NoSuchBeanDefinitionException, NoUniqueBeanDefinitionException;
+
+    Object getBean(String name) throws NoSuchBeanDefinitionException;
+
+    <T> T getBean(String name, Class<T> beanType) throws NoSuchBeanDefinitionException;
+
+    Map<String, Object> getBeans(Class<?> beanType);
+}
```

Comment:
Should each method throws NoUniqueBeanDefinitionException?

Reply:
we have Set for name, so, I suppose that not

Reply:
for 14-th row - suppose not too, because Name always unique